### ISSUE TITLE
Primary vesicoureteral reflux

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6821]
**Preferred term label:**
Primary vesicoureteral reflux

**Synonyms**

**Definition (free text, please give PubMed ID)**
Vesicoureteral reflux due to abnormalities in ureterovesical junction, e.g. ectopic insertion of the ureter or short intravesical tunnel at the ureterovesical junction.
PMID: 19668250, 18322164

**Parent term (use hpo.jax.org/app)**
Vesicoureteral reflux HP:0000076

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by RichardKlaus** [comment]
For all of the following VUR terms, I would suggest the following hierarchy

Vesicoureteral reflux HP:0000076
	Primary Vesicourtereal Reflux
	Secondary Vesicoureteral Reflux
	Low-Grade Ureteral Reflux
		Vesicoureteral Reflux Grade I
		Vesicoureteral Reflux Grade II
		Vesicoureteral Reflux Grade III
	High-Grade Vesicoureteral Reflux
		Vesicoureteral Reflux Grade IV
		Vesicoureteral Reflux Grade V
	Intrarenal reflux

**Comment by pnrobinson** [comment]
added
